cachi2/core/package_managers/pip.py

```diff
@@ -1917,8 +1957,8 @@ def _add_cachito_hash_to_url(parsed_url: urllib.parse.ParseResult, hash_spec: st
 
 
 def _to_checksum_info(hash_: str) -> ChecksumInfo:
-    algorithm, _, digest = hash_.partition(":")
-    return ChecksumInfo(algorithm, digest)
+    _, _, digest = hash_.partition(":")
```

Comment:
You can't do this, because then it's up to the runtime what it assigns to `_` and then use in the return line below, so this hunk needs to be dropped. This change wasn't previously here.

Reply:
Argh

Reply:
Fixed